prepare_inventory.py

```diff
@@ -0,0 +1,53 @@
+#!/bin/env python3
+
+# This is simple script which converts EDPM inventory from the OpenShift's secret
+# to the very simple inventory file which can be used by the RHOSO break/fix training
+# scripts while preparing training scenarios.
+# Usage of the script:
+# python3 prepare_inventory.py <path_to_the_edpm_inventory.yaml> <path_to_the_output_file.yaml>
+#
+# Edpm inventory file can be get from the OpenShift with command like:
+#
+# oc get secret <secret_name> -o json | jq '.data | map_values(@base64d)' | jq -r '.inventory'
+
+
+import yaml
+import sys
+
+
+if __name__ == '__main__':
+    if len(sys.argv) < 3:
+        print("Path to the yaml file with the RHOSO edpm inventory is "
+              "required to be passed as first argument and output file "
+              "path is required to be passed as second argument to this "
+              "script.")
+        sys.exit(1)
+
+    src_file = sys.argv[1]
+    dst_file = sys.argv[2]
+    if not src_file:
+        print("Yaml file with the RHOSO edpm inventory is required "
+              "to be passed as first argument.")
+        sys.exit(1)
+    if not dst_file:
+        print("Output file is required to be passed as second argument.")
+        sys.exit(1)
```

Comment:
No action required: Since this is a helper script which is used consistently, I'm fine with it as-is. But I'd consider an empty string as an invalid path just like providing a path to a non-existing file. That said, I think this should be enough:
```
try:
    src_file = sys.argv[1]
    dst_file = sys.argv[2]
except IndexError:
   print("You need 3 params blah blah")
   sys.exit(1)
```

Then later the L36 raises an error if a wrong path (including empty string) was provided